components/CommentsSection.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components'
+import CommentInput from './CommentInput'
+import Comment from './Comment'
+
+const Title = styled.h2`
+  font-size: 28px;
+  margin: 20px 10px;
+  font-family: ${(props) => props.theme.fontFamilyTitle};
+  @media screen and (min-width: 768px) {
+    margin: 40px 60px;
+  }
+`
+const testComment = "Comentario de Prueba - Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley"
+
+function CommentsSection({ comments = {} }) {
```

Comment:
The comments should be an array and should be iterated in the component, We're not using this component